README.md

```diff
@@ -133,8 +133,8 @@ You can find every supported distribution at the [official installation page](ht
 
 We encourage community contributions to Kong. To make sure it is a smooth
 experience (both for you and for the Kong team), please read
-[CONTRIBUTING.md](#CONTRIBUTING.md) , [DEVELOPER.md](#DEVELOPER.md),
-[CODE_OF_CONDUCT.md](#CODE_OF_CONDUCT.md) and [COPYRIGHT](#COPYRIGHT) before
+[CONTRIBUTING.md](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md) , [DEVELOPER.md](https://github.com/Kong/kong/blob/master/DEVELOPER.md),
+[CODE_OF_CONDUCT.md](https://github.com/Kong/kong/blob/master/CODE_OF_CONDUCT.md) and [COPYRIGHT](https://github.com/Kong/kong/blob/master/COPYRIGHT) before
```

Comment:
Please use this syntax instead, which will not force the `master` branch:

```suggestion
[CONTRIBUTING.md](CONTRIBUTING.md), [DEVELOPER.md](DEVELOPER.md),
[CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) and [COPYRIGHT](COPYRIGHT) before
```

See [the github docs](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files) for more details.

Reply:
Made the changes